client/src/app/collections/status-collection.coffee

```diff
@@ -0,0 +1,11 @@
+$ = require('jquery')
+Backbone = require('backbone')
+StatusModel = require('../models/status')
+
+module.exports = Backbone.Collection.extend(
+  model: StatusModel
+  comparator: (model) ->
+  	-model.get('id')
+  initialize: ->
+  	@sort(silent: true)
```

Comment:
Curious why we call `sort` manually there?  Since we defined a comparator, wont the collection stay sorted automatically as entries are added and removed?